### PR TITLE
Additional layers.

### PR DESCRIPTION
**Pull request by fxi**
First pull request, starting with something small :)

In markers object, the user can add an additional leaflet layer:

``` js
balkash: {lat: 44.9117998, lon: 74.1202449, zoom: 12, layer:layers['test']},
```

Test : http://fxi.io/storymap

Thanks

### CODE REVIEW

File: storymap.js [hunk 1]
```diff
@@ -113,6 +113,10 @@
                     map.setView(initPoint, initZoom, true);
                 } else if (markers[key]) {
                     var marker = markers[key];
+                    var layer = marker.layer;
+                    if(typeof layer!='undefined'){
```
Comment (atlefren) [inline storymap.js]: add some more spaces here:  if (typeof layer != 'undefined') {
and, use double unequality: typeof layer !== 'undefined'

**Comment by atlefren** [comment]
Cool! Looks good to me, some comments inline if you could fix them?

**Comment by fxi** [comment]
I've done the changes. It was more an exercise than a real pull request, feel free to refuse this commit :) 
Thanks

**Comment by atlefren** [comment]
:) This adds functionality without making it more complicated, so I see no reason not to merge it! 

(btw: out of curiosity: are you using this lib for someting, or just playing around?)

**Comment by fxi** [comment]
It was a mainly a request from a student who makes an internship where I work (University of Geneva/ UNEP). I've not much info about her project. 

Maybe, in a near future, I will write a small [R-leaflet](https://rstudio.github.io/leaflet/) extension in R to facilitate story maps creation in [Shiny](http://shiny.rstudio.com/gallery/). It's a bit strange to use R for this kind of project, but there is a great interest in presenting modelling or scientific results on interactive maps.  And I appreciate KISS approach, so storymap.js is really interesting. 

Thanks again !

**Pull request merged**